planning/obstacle_stop_planner/include/obstacle_stop_planner/planner_data.hpp

```diff
@@ -86,8 +87,14 @@ struct NodeParam
   // voxel grid z parameter for filtering pointcloud [m]
   double voxel_grid_z;
 
+  // It uses only predicted objects for slowdown and collision checking
   bool use_predicted_objects;
 
+  // If use_predicted_objects is true, objects are ignored if distance to trajectory is larger than
+  // this value [m]
+  double predicted_object_filtering_threshold;
```

Comment:
Please add this param in `obstacle_stop_planner` config in universe.

https://github.com/autowarefoundation/autoware.universe/blob/52127f0e7849a15928db370b99b5e2dcfa5b5e52/planning/obstacle_stop_planner/config/obstacle_stop_planner.param.yaml#L9-L13